src/components/ShowOptions/show-options.css

```diff
@@ -1,7 +1,11 @@
+.show-options-container{
+    position: relative;
+}
+
 .show-list{
     display: flex;
     position: absolute;
-    bottom: 0rem;
+    bottom: -1;
```

Comment:
you could mention the unit if its not zero. otherwise it might behave abnormal

Reply:
yes, made the changes, thank you for pointing out.